virttest/libvirt_remote.py

```diff
@@ -0,0 +1,42 @@
+"""
+Shared code for tests that need to control libvirt on remote
+"""
+
+import logging
+
+from avocado.core import exceptions
+
+from virttest import remote
+
+from virttest.utils_test import libvirt
+
+
+def update_remote_config(params, new_key,
```

Comment:
It seems this function can apply to any file on remote host, instead of configuration file only. So could you help update it for common file?

Reply:
@dzhengfy There is an action to restart service at end of this func, so I suggest only for configuration file for now.
Or there is any other files related to libvirt but not config files? Please let me know if I'm wrong.

Reply:
@Yingshun , I just think it may helpful in case some scenarios need update a remote file. For libvirtd restarting, we can add a new parameter to indicate if restarting is needed. What do you think of?

Reply:
@dzhengfy Okay, will update